src/main/scala/cromwell/server/WorkflowManagerSystem.scala

```diff
@@ -0,0 +1,15 @@
+package cromwell.server
+
+import akka.actor.ActorSystem
+import cromwell.engine.WorkflowManagerActor
+
+trait WorkflowManagerSystem {
+  val systemName = "cromwell-system"
+  val actorSystem = ActorSystem(systemName)
+
+  actorSystem.registerOnTermination({
+    actorSystem.log.info(s"$systemName shutting down")
+  })
```

Comment:
No real objection.  If you assign it back to me when you're ready I'll merge it.